conda/plugins/manager.py

```diff
@@ -175,6 +175,14 @@ def yield_command_hook_actions(self, hook_type: CommandHookTypes, command: str):
             if command in command_hook.run_for:
                 yield command_hook.action
 
+    def disable_external_plugins(self) -> None:
+        """
+        Disables all currently registered plugins except built-in conda plugins
+        """
+        for name, _ in self.list_name_plugin():
+            if not name.startswith("conda.plugins"):
```

Comment:
```suggestion
            if not name.startswith("conda."):
```
Let's not presume that all plugins will forever live in the `conda.plugins` package.